test/any/Canonicalizer-test.js

```diff
@@ -0,0 +1,181 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+describe('Canonicalizer', function() {
+  var Canonicalizer;
+  var IDLFile;
+  var IDLFileContents;
+  var Parser;
+
+  beforeEach(function() {
+    Canonicalizer = foam.lookup('org.chromium.webidl.Canonicalizer');
+    IDLFile = foam.lookup('org.chromium.webidl.IDLFile');
+    IDLFileContents = foam.lookup('org.chromium.webidl.IDLFileContents');
+    Parser = foam.lookup('org.chromium.webidl.Parser');
+  });
+
+  it('should return same number of files since we have different interfaces', function(done) {
+    // Setting up files for canonicalization.
```

Comment:
Consider writing a helper that simply receives a string containing IDL with exactly one top-level entity and returns the string.